acquisitions/src/main/resources/thunderjet/consortia/features/pieces-api-test-for-cross-tenant-envs.feature

```diff
@@ -0,0 +1,498 @@
+    * def v = login universityUser1
+    * table groupDetails
+      | id            | group   | okapitenant  |
+      | patronGroupId | 'staff' | targetTenant |
+    * def v = call createUserGroup groupDetails
+
+    # 1.3 Set patron group for central user
+    * def v = login consortiaAdmin
+    * configure headers = centralHeaders
+    * table userGroupDetails
+      | userId         | groupId       |
+      | centralAdminId | patronGroupId |
+    * def v = call setUserPatronGroup userGroupDetails
+
+    # 2. Setup Circulation Policy
+    * def v = call read('classpath:thunderjet/mod-orders/reusable/create-circulation-policy.feature')
+    * configure headers = targetHeaders
+    * def v = call read('classpath:thunderjet/mod-orders/reusable/create-circulation-policy.feature')
+    * configure headers = centralHeaders
+
+    # 3.1 Create piece for central
+    * def centralPieceId = call uuid
+    * set minimalPiece.id = centralPieceId;
+    * set minimalPiece.titleId = titleId;
+    * set minimalPiece.poLineId = poLineId;
+    * set minimalPiece.locationId = centralLocationsId;
+    Given path 'orders/pieces'
+    And param createItem = true
+    And request minimalPiece
+    When method POST
+    Then status 201
+    And match response.id == '#(centralPieceId)'
+    And match response.poLineId == '#(poLineId)'
+    And match response.titleId == '#(titleId)'
+    And match response.itemId == '#present'
+    And match response.holdingId == '#present'
+    And def itemId1 = response.itemId
+
+    # 3.2 Create piece for university
+    * def universityPieceId = call uuid
+    * set minimalPiece.id = universityPieceId;
+    * set minimalPiece.titleId = titleId;
+    * set minimalPiece.poLineId = poLineId;
+    * set minimalPiece.locationId = universityLocationsId;
+    * set minimalPiece.receivingTenantId = universityTenant;
+    Given path 'orders/pieces'
+    And param createItem = true
+    And request minimalPiece
+    When method POST
+    Then status 201
+    And match response.id == '#(universityPieceId)'
+    And match response.poLineId == '#(poLineId)'
+    And match response.titleId == '#(titleId)'
+    And match response.itemId == '#present'
+    And match response.holdingId == '#present'
+    And def itemId2 = response.itemId
+
+    # 4.1 Create Circulation Requests
+    * def requestId1 = call uuid
+    * def requestId2 = call uuid
+
+    * table request1
+      | id         | userId         | itemId  | holdingId         | instanceId
+      | requestId1 | centralAdminId | itemId1 | centralHoldingId1 | centralInstanceId
+    * def v = call createCirculationRequest request1
+    * call pause 1000
+
+    * configure headers = targetHeaders
+    * table request2
+      | id         | userId         | itemId  | holdingId            | instanceId
+      | requestId2 | centralAdminId | itemId2 | universityHoldingId1 | universityInstanceId
+    * def v = call createCirculationRequest request2
+
+    # 4.2 Verify circulation request 2
```

Comment:
is it formatted correctly, because it differs from formatting in this file (text in red)

Reply:
Looks good in the IDE, probably missing `|` at the end of the lines. Will add

Reply:
Should be resolved now